setup.py

```diff
@@ -32,6 +32,9 @@ def read_file(name):
         'Programming Language :: Python :: 3.4',
```

Comment:
`setup.py` can be removed once poetry with `pyproject.toml` is in place

Same for `setup.cfg` and `MANIFEST.in`

Reply:
deleted these too